src/BlockchainTestsFiller/ValidBlocks/bcEIP3675/NoBlockRewardKeepTipFiller.yml

```diff
@@ -0,0 +1,121 @@
+# Check the effects of maxPriorityFeePerGass on the miner's (coinbase) balance
+
+
+NoBlockRewardKeepTip:
+  genesisBlockHeader:
+    bloom: '0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000'
+    coinbase:  0xBA5E000000000000000000000000000000000000
+    difficulty: '131073'
+    extraData: '0x42'
+    gasUsed: '0'
+    mixHash: '0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421'
+    nonce: '0x0102030405060708'
+    number: '0'
+    parentHash: '0x0000000000000000000000000000000000000000000000000000000000000000'
+    receiptTrie: '0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421'
+    stateRoot: '0xf99eb1626cfa6db435c0836235942d7ccaa935f1ae247d3f1c21e495685f903a'
+    timestamp: '0x03b6'
+    transactionsTrie: '0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421'
+    uncleHash: '0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347'
+    gasLimit: 10000000000
+    gasUsed:   10000000000
+    baseFeePerGas: 1000
+  sealEngine: NoProof
+
+  pre:
+  # Extremely simple (and cheap in gas) contract. Just a push, 3 gas
+    0xaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa:
+      balance: 0x10000000000
+      nonce: 1
+      code: :raw 0x60FF00
+      storage: {}
+
+    0xd02d72E067e77158444ef2020Ff2d325f929B363:
+      balance: 0x10000000000000000
+      nonce: 1
+      code: ''
+      storage: {}
+
+    #Store the beneficiary miner balance (coinbase) after blocks mined.
+    0xcccccccccccccccccccccccccccccccccccccccc:
+      balance: 0x10000000000
+      nonce: 1
+      code: |
+         :yul { sstore(sub(number(),1), balance(coinbase()) ) }
+      storage: #See if storage is set back to 0 for first two blocks.
+          0x01: 1
```

Comment:
what about block numbers 3,4 ?

Reply:
I have added a default storage slot value for block 3. 
After block 4 is mined, it will show the balance after block three was mined in storage slot 0x03.